packages/react-app/src/components/pages/Bounties/Form/FormFields.tsx

```diff
@@ -0,0 +1,157 @@
+import { Controller, UseFormReturn } from 'react-hook-form';
+import React from 'react';
+import DatePicker from '@app/components/parts/DatePicker';
+import {
+	Box,
+	FormLabel,
+	FormControl,
+	Input,
+	FormErrorMessage,
+	Select,
+	Flex,
+	FormControlProps,
+	Textarea,
+} from '@chakra-ui/react';
+import { dateIsNotInPast, required, validNonNegativeDecimal } from '@app/utils/formUtils';
+
+const useCurrencies = (): string[] => {
+	return ['BANK'];
```

Comment:
We don't have a multitenant approach on the bot for allowlisted currencies per customer, so our master list to save you a trip: 

```
const ALLOWED_CURRENCIES = ['BANK', 'ETH', 'BTC', 'USDC', 'USDT', 'TempCity', 'gOHM', 'LUSD', 'FOX', 'oneFOX'];
```